src/metabase/email/messages.clj

```diff
@@ -596,10 +596,13 @@
 (defn- alert-context
   "Context that is applicable only to the actual alert template (not alert management templates)"
   [alert channel non-user-email]
-  (let [{card-id :id, card-name :name} (first-card alert)]
+  (let [{card-id :id card-name :name} (first-card alert)]
     {:title                     card-name
      :titleUrl                  (urls/card-url card-id)
      :alertSchedule             (alert-schedule-text channel)
+     :notificationText          (if (nil? non-user-email)
```

Comment:
this was a bug before, alert email for non-user doesn't have the manage subscriptions link.